op-service/txmgr/txmgr.go

```diff
@@ -213,38 +213,41 @@ func (m *SimpleTxManager) send(ctx context.Context, candidate TxCandidate) (*typ
 		ctx, cancel = context.WithTimeout(ctx, m.cfg.TxSendTimeout)
 		defer cancel()
 	}
-	res, err := m.daClient.SubmitPFB(ctx, m.namespaceId, candidate.TxData, 20000, 700000)
-	if err != nil {
-		m.l.Warn("unable to publish tx to celestia", "err", err)
-		return nil, err
-	}
-	fmt.Printf("res: %v\n", res)
+	if candidate.To.Hex() == "0xfF00000000000000000000000000000000000000" {
```

Comment:
Add a comment to clarify why we check for this address

Reply:
Added explanation as a TODO note.